### PR TITLE
Update BT/BQ paths.

### PR DESCRIPTION
**Pull request by Spaceenter**

### CODE REVIEW

File: deployment/staging_bq_dataset.txt [hunk 1]
```diff
@@ -1 +1 @@
-google.com:datcom-store-dev.dc_kg_2020_06_30_01_02_51
\ No newline at end of file
+google.com:datcom-store-dev:dc_kg_2020_07_07_19_47_23
```
Comment (shifucun) [inline deployment/staging_bq_dataset.txt:1]: the second colon should be "."

Comment (tjann) [reply deployment/staging_bq_dataset.txt:1]: String check here? ;)

Comment (shifucun) [reply deployment/staging_bq_dataset.txt:1]: good point. This will fail the e2e sparql test as well

**Comment by shifucun** [comment]
Can you also update the image to "gcr.io/datcom-mixer/go-grpc-mixer:0.10.0"

**Pull request closed without merging**